Blocks/HTTP/impl/posix_server.h

```diff
@@ -197,6 +199,51 @@ class HTTPServerPOSIX final {
   }
 
  private:
+  void FindHandler(const std::string& path,
+                   std::function<void(Request)>& dest_handler,
+                   URLPathParams& dest_path_params) {
+    assert(!path.empty());
+    assert(path[0] == '/');
+    std::string& remaining_path = dest_path_params.base_path;
+    remaining_path = path;
+
+    auto MatchPathParamCounts =
```

Comment:
Done in https://github.com/c5t/Current/commit/91cd01b19758c0a3e67b3f5be4d77caf10213bd6